src/Button/DigitizeButton/DigitizeButton.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { Modal, Input } from 'antd';
+import Modal from 'antd/lib/modal';
+import Input from 'antd/lib/input';
```

Comment:
Are you sure, that we should adjust `antd` imports by hand?

I thought, that's exactly the task of `babel-plugin-import`..

=> https://ant.design/docs/react/getting-started#Import-on-Demand

Reply:
And why do we import from `lib` here and from `es` in the babel config?

Reply:
we are trying to stick to an unified way of importing modules so we though why not do it for all libraries. But for some reason when importing from `antd` the plugin reduces the size much more than just the individual module import.

Reply:
@dnlkoch because the `es` folder made some tests fail and we couldn't find the exact cause so we opted for the easiest solution. However, I would like to have the `antd` imported with the plugin since it is more effective but it is also not good to have two different patterns for different libraries.

Reply:
yes it fixes the tests as well.

Reply:
No, no need for that. Just ignore it!

Reply:
To clear this up, it was my idea to have all imports harmonized and in `the right way`™

The main problems we currently face in this PR have to do with the wrong way we and other libraries use the import statements.

I know that the plugin can repair our somewhat broken kind of imports, but i thought it would be best practice to import correctly everywhere and avoid mixed imports.
Going that way would have reduced the risk of doing imports wrong in the future.

If there was enough time, it would be best to examine what exactly the difference is when using the plugin vs. explicit imports, but i guess this has to be ok for the moment ... :-)